### PR TITLE
vkcube: Implement key events in Windows

### LINKED ISSUE #42: Cube on Windows does not respond to keyboard commands
Cube responds to keys "space", "Esc", "left arrow", and "right arrow".  I believe it used to do this on Windows as well, but it no longer does.

Please add/re-add the keyboard command functionality on Windows.

### PR DESCRIPTION
**Pull request by krOoze**
close #42

### CODE REVIEW

File: cube/cube.cpp [hunk 1]
```diff
@@ -2944,6 +2960,13 @@ int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPSTR pCmdLine,
 
     // main message loop
     while (!done) {
+        if(demo.pause) {
+            const BOOL succ = WaitMessage();
+
+            const auto& suppress_popups = demo.suppress_popups;
```
Comment (jeremy-lunarg) [inline cube/cube.cpp]: Do you need suppress_popups?

Comment (krOoze) [reply cube/cube.cpp]: The `ERR_EXIT` unfortunately needs it.

Comment (jeremy-lunarg) [reply cube/cube.cpp]: ah, yes, thanks

**Comment by krOoze** [comment]
Ups, forgot clang-format (again).

**Comment by Tony-LunarG** [comment]
LGTM

**Pull request merged**